modules/couchbase/charts.go

```diff
@@ -0,0 +1,76 @@
+package couchbase
+
+import (
+	"github.com/netdata/go.d.plugin/agent/module"
+)
+
+type (
+	Charts = module.Charts
+	Chart  = module.Chart
+	Dim    = module.Dim
+)
+
```

Comment:
i suggest to create a var `bucketCharts`, so you can

```go
   return bucketCharts.Copy(), nil
```

in `initCharts()`

Like

```go
var bucketCharts = module.Charts{
	dbPercentCharts.Copy(),
    ...
}
```